build.gradle

```diff
@@ -246,7 +246,8 @@ ext.externalDependency = [
   "reflections" : "org.reflections:reflections:0.9.10",
   "embeddedProcess": "de.flapdoodle.embed:de.flapdoodle.embed.process:1.50.2",
   "testMysqlServer": "com.wix:wix-embedded-mysql:1.1.0",
-  "flyway": "org.flywaydb:flyway-core:3.2.1"
+  "flyway": "org.flywaydb:flyway-core:3.2.1",
+  "oltu": "org.apache.oltu.oauth2:org.apache.oltu.oauth2.client:1.0.1"
```

Comment:
Is this version available in our internal artifactory? Why not using the latest version?